docs/fundamentals/dashboard/configuration.md

```diff
@@ -102,9 +102,37 @@ API key authentication works by requiring each OTLP request to have a valid `x-o
 |--|--|--|
 | `Dashboard:Otlp:AuthMode` | `Unsecured` | Can be set to `ApiKey`, `Certificate` or `Unsecured`. `Unsecured` should only be used during local development. It's not recommended when hosting the dashboard publicly or in other settings. |
 | `Dashboard:Otlp:PrimaryApiKey` | `null` | Specifies the primary API key. The API key can be any text, but a value with at least 128 bits of entropy is recommended. This value is required if auth mode is API key. |
-| `Dashboard:Otlp:SecondaryApiKey` | `null` | Specifies the secondary API key. The API key can be any text, but a value with at least 128 bits of entropy is recommended. This value is optional. If a second API key is specified then the incoming `x-otlp-api-key` header value can match either the primary or secondary key. |
+| `Dashboard:Otlp:SecondaryApiKey` | `null` | Specifies the secondary API key. The API key can be any text, but a value with at least 128 bits of entropy is recommended. This value is optional. If a second API key is specified, then the incoming `x-otlp-api-key` header value can match either the primary or secondary key. |
 
-### Resources
+## OTLP CORS
+
+CORS (Cross-Origin Resource Sharing) is a mechanism that allows many resources (for example, fonts, JavaScript, etc.) on a web page to be requested from another domain outside the domain from which the resource originated.
+
+With the .NET Aspire dashboard, the OTLP endpoint can be configured to allow cross-origin requests. This is useful when the dashboard is hosted on a different domain than the OTLP endpoint, and this can enable scenarios where browser telemetry is sent directly to the OTLP endpoint as demonstrated in the [browser telemetry](https://github.com/dotnet/aspire/tree/main/playground/BrowserTelemetry) sample.
+
+To configure CORS, use the `Dashboard:Otlp:Cors` section and specify the allowed origins and headers:
+
+```json
+{
+  "Dashboard": {
+    "Otlp": {
+      "Cors": {
+        "AllowedOrigins": [ "https://example.com" ],
+        "AllowedHeaders": [ "Content-Type" ]
+      }
+    }
+  }
+}
+```
+
+Consider the following configuration options:
+
+| Option | Default Value | Description |
+|--|--|--|
+| `DASHBOARD__OTLP__CORS__ALLOWEDORIGINS` | `null` | A comma-delimited list of allowed origins for CORS. This setting is optional and a shortcut to `Dashboard:Otlp:Cors:AllowedOrigins`. |
```

Comment:
It's not a shortcut. It's an alternative syntax that's required if the config is specified in an environment variable.

Please change this section to be consistent with all the others which have colons.

Reply:
The description should mention:
* The domain here should be the browser app that wants to send telemetry. Otherwise people could think they specify the dashboard domain
* `*` is a wildcard and allows any domain to send telemetry to the dashboard

Reply:
> It's not a shortcut. It's an alternative syntax that's required if the config is specified in an environment variable.

The shortcut phrasing was borrowed from another section in this doc for consistency. See the description of the `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL`. I'll update that section too.

> Please change this section to be consistent with all the others which have colons.

Not sure I follow, there are some sections that lead with the env var and others that lead with the `:` based config details. Happy to be consistent across all, but need to make sure I understand the ask here.

Reply:
> The shortcut phrasing was borrowed from another section in this doc for consistency. See the description of the `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL`. I'll update that section too. Also, do the env vars take precedence over the JSON-based config?

No, in the one case those configuration options are shortcuts. That's why they discuss the underlying config value that they set.

Reply:
Updated...